.github/workflows/build-and-test.yml

```
@@ -32,6 +32,6 @@ jobs:
     needs: [build_and_test-spm, build_and_test-uitests]
     uses: StanfordBDHG/.github/.github/workflows/create-and-upload-coverage-report.yml@v1
     with:
-      coveragereports: HealthKitOnOMH.xcresult HealthKitOnOMH.xcresult
+      coveragereports: HealthKitOnOMH-Package.xcresult
```

@vishnuravi If you also want to get any test coverage from the UI tests you would need to pass in the following coverage reports:
```suggestion
      coveragereports: HealthKitOnOMH-Package.xcresult TestApp.xcresult
```
Alternatively, if you just want to only use the unit tests (wich are sufficient for this package IMO) you can just directly use the CodeCov action without the need to merge two coverage reports: https://github.com/codecov/codecov-action

---

Thanks, I've removed the UI tests since they are not necessary here (as you noted)!